dashboard/src/components/js/Alert/Alert.js

```diff
@@ -0,0 +1,109 @@
+import React from "react";
+import AlerTypes from "./Alert.types";
+import PropTypes from "prop-types";
+import Button from "../Button";
+import cs from "classnames";
+import {
+  ClarityIcons,
+  infoCircleIcon,
+  exclamationCircleIcon,
+  checkCircleIcon,
+  exclamationTriangleIcon,
+  timesIcon,
+} from "@clr/core/icon-shapes";
+import { CdsIcon } from "../../Clarity/clarity";
```

Comment:
cc/ @Angelmmiguel note the new way to import icons. The react wrapper has been extracted (`../../Clarity/clarity`) from https://stackblitz.com/edit/react-ts-wrapper-5w8nhf. I'd assume that there is a similar wrapper for plain JS?

Reply:
Yeah, the approach is pretty similar, although they provide you some helpers to install a certain group of icons directly.

I'm curious about how importing the same icon in two different components affects to the library. Not sure about this use case since we load all the icons in a single file internally.